src/pages/Home/subcomponents/_CityMap.scss

```diff
@@ -382,6 +382,7 @@
         }
         @media screen and (max-width: 500px) {
           min-width: 350px;
+          transform: translateX(20px);
```

Comment:
@GreenWithMV I personally like using [`truncate`](https://css-tricks.com/snippets/css/truncate-string-with-ellipsis/) when dealing with minimum width as well, as it allows for pretty decent fallback options that prevent breaks of the page layout (and slightly nudge users to use better sized input). Good use of `translateX` as well!

Reply:
That is cool, I've never used text-overflow: ellipsis. I'll throw it on for the geocoder as that is a little more "on purpose" looking than the overflow that is happening now which looks like an error.